src/harness/reference_models/pre_iap_filtering/pre_iap_filtering.py

```diff
@@ -20,13 +20,15 @@
 """
 from reference_models.fss_purge import fss_purge
 from reference_models.inter_sas_duplicate_grant import inter_sas_duplicate_grant
+from reference_models.ppa_gwpz_fss_gwbl_purge import ppa_gwpz_fss_gwbl_purge
```

Comment:
Here I see the cyclical dependence between the 2 package. The modules (files) could perhaps be on the same directory.

Reply:
moved   the file to 'pre_iap_filtering' directoy